src/ansys/fluent/core/post_objects/post_objects.py

```diff
@@ -0,0 +1,156 @@
+"""Module providing visualization objects for Matplotlib."""
+import inspect
+import sys
+
+from ansys.fluent.core.meta import PyLocalContainer
+
+
+class Plots:
```

Comment:
Can we have Containter class and both Graphics and Plot can derive from that,

Reply:
yes, working on that currently.

Reply:
Done, Thank you